packages/react/src/analytics/integrations/Guardian/Guardian.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Guardian Integration. Loads the Guardian script and calls its methods.
+ *
+ * @example <caption>Adding Guardian integration to analytics</caption>
+ * ```
+ *
+ * import \{ analytics, integrations \} from '\@farfetch/blackout-react/analytics';
+ *
+ * analytics.addIntegration('guardian', integrations.Guardian, \{\});
+ * ```
+ */
+
+import {
+  EventData,
+  integrations,
+  LoadIntegrationEventData,
+  SetUserEventData,
+  StrippedDownAnalytics,
+  TrackTypesValues,
+  utils,
+} from '@farfetch/blackout-analytics';
+import { Forter, Riskified } from './providers';
+import {
+  getEnvironmentFromOptions,
+  getSafeOptionsForIntegration,
+} from './guardian-helper';
+import {
+  type GuardianIntegrationBaseOptions,
+  type GuardianIntegrationOptions,
+  GuardianProviders,
+} from './types';
+import omit from 'lodash/omit';
+import type { ForterIntegrationOptions } from './providers/Forter/types';
+import type { RiskifiedIntegrationOptions } from './providers/Riskified/types';
+
+/**
+ * Guardian integration.
+ */
+class Guardian extends integrations.Integration<GuardianIntegrationOptions> {
+  forterInstance?: Forter;
+  riskifiedInstance?: Riskified;
+  baseOptions: GuardianIntegrationBaseOptions;
+  activeIntegrations: Array<string>;
+
+  /**
+   * Creates an instance of Guardian integration.
+   *
+   * @param options   - Integration options.
+   * @param loadData  - Analytics' load event data.
+   * @param analytics - Analytics stripped down instance.
+   */
+  constructor(
+    options: GuardianIntegrationOptions = {},
+    loadData: LoadIntegrationEventData,
+    analytics: StrippedDownAnalytics,
+  ) {
+    const baseOptions: GuardianIntegrationBaseOptions = omit(options, [
+      'riskified',
+      'forter',
+    ]);
+    super(options, loadData, analytics);
+
+    this.baseOptions = baseOptions;
+
+    baseOptions.environment = getEnvironmentFromOptions(options);
+
+    // load default if not set
+    this.activeIntegrations = baseOptions.activeIntegrations || [
+      GuardianProviders.Forter,
+      GuardianProviders.Riskified,
+    ];
+
+    // Check if activeIntegrations is an array
+    if (!Array.isArray(this.activeIntegrations)) {
+      utils.logger.error(
+        '[Analytics] Guardian - The value `activeIntegrations` from Guardian integration options must be an array.',
```

Comment:
Also be consistent with the error messages. Other integrations do not prefix `[Analytics]` to their error messages and just prefix the integration by itself like `[Guardian]`. Take a look at other integrations error messages and use the same format, please.

Reply:
Fixed